ui/mirage/config.js

```diff
@@ -118,7 +118,69 @@ export default function () {
         let nextToken = req.queryParams.next_token || 0;
         let reverse = req.queryParams.reverse === 'true';
         const json = this.serialize(jobs.all());
-        let sortedJson = json
+
```

Comment:
Just enough filter logic to let us test things reasonably. But no more than that.